text/processing.py

```diff
@@ -121,11 +122,11 @@ class ConcatenateJob(Job):
 
     __sis_hash_exclude = {"zip_out": True, "out_name": "out"}
 
-    def __init__(self, text_files, zip_out=True, out_name="out"):
+    def __init__(self, text_files: List[Path], zip_out: bool = True, out_name: str = "out"):
```

Comment:
Is List required here? Or would Iterable suffice?

Reply:
Not sure if all iterables support `len()`, otherwise everything could be written such that it works for any iterable I guess. I just used list because it was annotated like this before.